kong/conf_loader/init.lua

```diff
@@ -72,244 +72,24 @@ ffi.cdef([[
-                 dhparams = "ffdhe2048",
-    prefer_server_ciphers = "off",
-  },
-                      old = {
-                protocols = "TLSv1 TLSv1.1 TLSv1.2 TLSv1.3",
-                  ciphers = "ECDHE-ECDSA-AES128-GCM-SHA256:"
-                         .. "ECDHE-RSA-AES128-GCM-SHA256:"
-                         .. "ECDHE-ECDSA-AES256-GCM-SHA384:"
-                         .. "ECDHE-RSA-AES256-GCM-SHA384:"
-                         .. "ECDHE-ECDSA-CHACHA20-POLY1305:"
-                         .. "ECDHE-RSA-CHACHA20-POLY1305:"
-                         .. "DHE-RSA-AES128-GCM-SHA256:"
-                         .. "DHE-RSA-AES256-GCM-SHA384:"
-                         .. "DHE-RSA-CHACHA20-POLY1305:"
-                         .. "ECDHE-ECDSA-AES128-SHA256:"
-                         .. "ECDHE-RSA-AES128-SHA256:"
-                         .. "ECDHE-ECDSA-AES128-SHA:"
-                         .. "ECDHE-RSA-AES128-SHA:"
-                         .. "ECDHE-ECDSA-AES256-SHA384:"
-                         .. "ECDHE-RSA-AES256-SHA384:"
-                         .. "ECDHE-ECDSA-AES256-SHA:"
-                         .. "ECDHE-RSA-AES256-SHA:"
-                         .. "DHE-RSA-AES128-SHA256:"
-                         .. "DHE-RSA-AES256-SHA256:"
-                         .. "AES128-GCM-SHA256:"
-                         .. "AES256-GCM-SHA384:"
-                         .. "AES128-SHA256:"
-                         .. "AES256-SHA256:"
-                         .. "AES128-SHA:"
-                         .. "AES256-SHA:"
-                         .. "DES-CBC3-SHA",
-    prefer_server_ciphers = "on",
-  },
-                     fips = { -- https://wiki.openssl.org/index.php/FIPS_mode_and_TLS
-                          -- TLSv1.0 and TLSv1.1 is not completely not FIPS compliant,
-                          -- but must be used under certain condititions like key sizes,
-                          -- signatures in the full chain that Kong can't control.
-                          -- In that case, we disables TLSv1.0 and TLSv1.1 and user
-                          -- can optionally turn them on if they are aware of the caveats.
-                          -- No FIPS compliant predefined DH group available prior to
-                          -- OpenSSL 3.0.
-                protocols = "TLSv1.2",
-                  ciphers = "TLSv1.2+FIPS:kRSA+FIPS:!eNULL:!aNULL",
-    prefer_server_ciphers = "on",
-  }
-}
-
-
-local DEFAULT_PATHS = {
-  "/etc/kong/kong.conf",
-  "/etc/kong.conf",
-}
+local CIPHER_SUITES = constants.CIPHER_SUITES
+local DEFAULT_PATHS = constants.DEFAULT_PATHS
+local HEADER_KEY_TO_NAME = constants.HEADER_KEY_TO_NAME
+local UPSTREAM_HEADER_KEY_TO_NAME = constants.UPSTREAM_HEADER_KEY_TO_NAME
+local DYNAMIC_KEY_NAMESPACES = constants.DYNAMIC_KEY_NAMESPACES
+local DEPRECATED_DYNAMIC_KEY_NAMESPACES = constants.DEPRECATED_DYNAMIC_KEY_NAMESPACES
+local PREFIX_PATHS = constants.PREFIX_PATHS
+local CONF_PARSERS = constants.CONF_PARSERS
```

Comment:
Just searched in this page, seem `CONF_PARSERS` is not used anywhere. Shall we just remove this line. 

Same for the `CONF_BASIC`, `HEADERS`, `DEPRECATED_DYNAMIC_KEY_NAMESPACES`, `DYNAMIC_KEY_NAMESPACES` , `UPSTREAM_HEADER_KEY_TO_NAME` , `HEADER_KEY_TO_NAME`, `DEFAULT_PATHS`.

Reply:
Ignore above comment. These variables are used (they are fold in the default view).